ci.override.env

```diff
@@ -4,4 +4,8 @@ USER_SERVICE_BRANCH=develop
 
 ROOM_SERVICE_PATH=./services/room-service
 ROOM_SERVICE_REPO=https://github.com/book-em/room-service.git
-ROOM_SERVICE_BRANCH=feature-create-reservation-requests
\ No newline at end of file
+ROOM_SERVICE_BRANCH=feature-create-reservation-requests
+
```

Comment:
Try setting this to `develop`, `feature-create-reservation-requests` is a pretty old branch and that's why the integration tests fail.